packages/stratocacher-layer-localstorage/.npmignore

```diff
@@ -0,0 +1,5 @@
+src/
+lib/dev/
+test/
+gulpfile.babel.js
+.* # dot files
```

Comment:
I think this might be broader than you intend...

https://git-scm.com/docs/gitignore

```
dougwade main/corvair ‹hackathon_spring_2016_s› » ls -a
.   .DS_Store	.gitignore  .settings	 cacheit.js  common	 corvair      doc	 gulpfile.js  newrelic.js   package.json  pom.xml    sites  start.js  target-entrypoints
..  .classpath	.project    build_tools  cluster     components  corvair.iml  dojohacks  middleware   node_modules  pages	  server.js  src    target    test
dougwade main/corvair ‹hackathon_spring_2016_s› » ls .settings
org.eclipse.jdt.core.prefs  org.eclipse.wst.common.component  org.eclipse.wst.common.project.facet.core.xml
dougwade main/corvair ‹hackathon_spring_2016_s› » ls .*
.DS_Store  .classpath  .gitignore  .project

.settings:
org.eclipse.jdt.core.prefs  org.eclipse.wst.common.component  org.eclipse.wst.common.project.facet.core.xml
```